<p>If should have else.</p>

<h2>Noncompliant Code Example</h2>
<pre>
  if(something) {
  do something
  }
</pre>

<h2>Compliant Solution</h2>
<pre>
  if(something) {
  do something
  } else {
  do this instead
  }
</pre>
