<p>Variables Should be Final</p>

<h2>Noncompliant Code Example</h2>
<pre>

  int test = 0;

  String s = "test";

</pre>

<h2>Compliant Solution</h2>
<pre>

  final int test = 0;

  final String s = "test";

</pre>
