<p>While and For Loops are not allowed</p>

<h2>Noncompliant Code Example</h2>
<pre>
Everything with while or for loops
</pre>

<h2>Compliant Solution</h2>
<pre>
Everything without while or for loops
</pre>
